pyproject.toml

```diff
@@ -26,3 +24,9 @@ exclude = '''
 profile = "black"
 combine_as_imports = true
 skip =  ["venv", "benchmarks"]
+
+[build-system]
+requires = ["setuptools>=45", "setuptools_scm[toml]>=6.2", "wheel"]
```

Comment:
This is a fundamental part of a `pyproject.toml` file, and is traditionally the very first section. Could you move it to the top ?